src/theme.ts

```diff
@@ -12,19 +12,22 @@ const palette = {
     // green colors
     main: "#006778",
     light: "#c2dbdf",
+    lighter: "#c2dbdf",
     lightest: "#e5eff1",
   },
   secondary: {
     // grey colors
     main: "#595959",
     light: "#dfe1e3",
+    lightest: "#DFE1E3",
```

Comment:
Do the `light` and `lightest` have the same colour ?

Reply:
Also this should be fixed now.